presto-hive/src/main/java/com/facebook/presto/hive/metastore/glue/GlueHiveMetastoreConfig.java

```diff
@@ -79,4 +80,17 @@ public GlueHiveMetastoreConfig setDefaultWarehouseDir(String defaultWarehouseDir
         this.defaultWarehouseDir = Optional.ofNullable(defaultWarehouseDir);
         return this;
     }
+
+    public Optional<String> getCatalogId()
+    {
+        return catalogId;
+    }
+
+    @Config("hive.metastore.glue.catalogid")
```

Comment:
nit: `hive.metastore.glue.catalog-id`?

Reply:
I used the same name that Amazon uses in their implementation: https://docs.aws.amazon.com/emr/latest/ReleaseGuide/emr-presto-glue.html#emr-presto-glue-configure

What do you think?

Reply:
I used the same name that Amazon uses in their implementation: https://docs.aws.amazon.com/emr/latest/ReleaseGuide/emr-presto-glue.html#emr-presto-glue-configure

What do you think?

Reply:
Oh thanks for the pointer. I'm not aware of that :). Then it might be good for us to be consistent.